value/valuereflect_test.go

```diff
@@ -560,6 +561,18 @@ func TestReflectList(t *testing.T) {
 			if !reflect.DeepEqual(iterateResult, tc.expectedIterate) {
 				t.Errorf("expected iterate to produce %#v but got %#v", tc.expectedIterate, iterateResult)
 			}
+
+			iter := m.Range()
+			iter.Recycle()
```

Comment:
is this supposed to be a defer?

Reply:
Yikes, yes. I'll review all the other Recycle calls too.